src/com/nutrons/framework/controllers/ServoController.java

```diff
@@ -0,0 +1,17 @@
+package com.nutrons.framework.controllers;
+
+import io.reactivex.functions.Consumer;
+
+public abstract class ServoController implements Consumer<ServoCommand> {
```

Comment:
Why's this class need to be abstract? Don't see any other subclasses besides `RevServo`